cfbs.json

```
@@ -139,6 +139,52 @@
       "subdirectory": "management/every-minute",
       "steps": ["json def.json def.json"]
     },
+    "file-permissions": {
+      "description": "Manages file permissions",
+      "tags": ["management"],
+      "repo": "https://github.com/nickanderson/cfengine-file-permissions",
+      "by": "https://github.com/nickanderson",
+      "version": "0.0.1",
+      "commit": "a678160011b834557f1ab42a081ea84adf8622df",
+      "steps": [
+        "copy file-permissions.cf services/file-permissions.cf",
+        "input file-permissions/input.json def.json",
+        "bundles nickanderson:file_permissions",
+        "json inputs-def.json def.json"
```

With the latest cfbs, you should be able to do:

```suggestion
        "policy_files services/file-permissions.cf"
```

(And delete your inputs-def.json from your repo)

---

Add in `services/cfbs/modules/file-permissions/file-permissions.cf` if you end up changing the path above.

---

policy_files with no source specified?